nfsstat/tests/test_nfsstat.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import mock
+import pytest
+from datadog_checks.stubs import aggregator as _aggregator
```

Comment:
if you just import this in the function it won't require an alias

Reply:
It's slower that way though

Reply:
You're only importing it a handful of times in any test, but your call